Cargo.toml

```diff
@@ -39,7 +39,12 @@ std = []
 kv = []
 kv_sval = ["kv", "value-bag/sval", "sval", "sval_ref"]
 kv_std = ["std", "kv", "value-bag/error"]
-kv_serde = ["kv_std", "value-bag/serde", "serde"]
+kv_serde = ["kv_std", "value-bag/serde", "serde_core"]
+
+# This is here to not break backwards compatibility with the implicit feature
+# that enables support for serde based on the dependency name. Since we're now
+# using serde_core, the implicit feature was renamed, this adds back an alias.
+serde = ["serde_core"]
```

Comment:
Also see the failed CI: https://github.com/rust-lang/log/actions/runs/19773284438/job/56661471080?pr=712.